applications/frontier-pos-template.md

```diff
@@ -0,0 +1,184 @@
+  https://github.com/substrate-developer-hub/substrate-node-template， No EVM compatibility and no NPOS.
+  https://github.com/substrate-developer-hub/frontier-node-template, No NPOS and the code structure is not developer-friendly.
+  https://github.com/paritytech/polkadot-sdk/tree/master/substrate/bin/node, No EVM and the code structure is not developer-friendly.
+  
+  Why is it necessary to develop this project even with the existing templates?   
+  We know that Polkadot's official team has focused more on innovations in the underlying technology. 
+  This is akin to creating airplanes to connect two cities, but once people leave the airport, 
+  they still need more convenient transportation like taxis or subways. This project serves a similar purpose. 
+  It greatly facilitates developers by eliminating the need to repeatedly perform the same tasks, 
+  making the development process much more enjoyable and efficient. It also helps more enthusiasts of NPOS and EVM understand how Substrate can be compatible with EVM.
+  This is very meaningful. In addition, as a public template, it will pay attention to the development of other EVM chains in the Polkadot ecosystem, 
+  pay attention to what interesting functions and RPC they have done, and provide it as much as possible. 
+  This will directly benefit more developers. These tasks cannot be complete directly by Polkadot officials, 
+  because they have their own areas of concern, which is the innovation of underlying technologies.
+
+  
+## Team :busts_in_silhouette:
+
+### Team members
+
+- weimeme
+- JimYam
+
+### Contact
+
+- **Contact Name:** weimeme
+- **Contact Email:** weimeme067@gmail.com
+
+### Legal Structure
+
+- **Registered Address:** Address of your registered legal entity, if available. Please keep it in a single line. (e.g. High Street 1, London LK1 234, UK)
+- **Registered Legal Entity:** N/A
+
+### Team's experience
+
+- `JimYam` is an active and enthusiastic developer in the Polkadot ecosystem, a code contributor since 2019. He has been guiding early developers in China and recommending them to apply for grants.
+
+- `weimeme` has nearly two years of development experience with Substrate and seven years with Ethereum. He previously worked at the Ethereum cross-chain bridge Orbiter and is very proficient in L2 and Ethereum technologies.
+
+### Team Code Repos
+
+- https://github.com/ChainSupport/frontier-pos-template
+
+- https://github.com/JimYam
+- https://github.com/weimeme
+
+> Because our entire team of developers typically serves different public chain development teams, 
+> there is a requirement for a certain level of anonymity for our past GitHub accounts. If you need to know which specific projects we participated in, feel free to message me privately.
```

Comment:
Can you please send us proof of your previous open source work to grants@web3.foundation?